challenge-manager/edge-tracker/tracker.go

```diff
@@ -379,10 +408,6 @@ func (et *Tracker) uniqueTrackerLogFields() []any {
 }
 
 func (et *Tracker) tryToConfirmEdge(ctx context.Context) (bool, error) {
-	// If the edge is not a root, block challenge edge, we have nothing to do here.
```

Comment:
I think we still want to check that the edge has a claim id (that is, it is a "root" edge). We don't care about confirming other type of edges unless they are one step provable.

Reply:
Done.